composer.json

```diff
@@ -66,9 +66,6 @@
             "Tests/"
```

Comment:
Does it mean that the classmap is not used in dev? (That would make sense, but I'm unsure). If yes, your PR is indeed ok.